src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -290,6 +293,7 @@ function MoneyRequestView({
                     onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_DISTANCE.getRoute(CONST.IOU.ACTION.EDIT, iouType, transaction?.transactionID ?? '', report.reportID))}
                 />
             </OfflineWithFeedback>
+            {/* TODO: correct the pending field action https://github.com/Expensify/App/issues/36988 (?) */}
```

Comment:
Why the `(?)`? 😄

Reply:
I wasn't sure if it's a proper issue to link

Reply:
Oh yeah, it should be this issue I think. Next time you could ask us directly or look at the [P2P distance project board](https://github.com/orgs/Expensify/projects/106/views/3).
https://github.com/Expensify/App/issues/36987